File: Gixor-Backend/Gixor-Api/Gixor-Api-GitHub/src/main/java/com/cpy3f2/Gixor/Domain/GitHubUserDO.java
package com.cpy3f2.Gixor.Domain;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.Builder;
import lombok.Data;

/**
 * @author : simon
 * @description :
 * @last : 2024-11-01 00:37
 * Copyright (c) 2024. 保留所有权利。
 * @since : 2024-11-01 00:37
 */
@Data
@Builder
@JsonIgnoreProperties(ignoreUnknown = true)
public class GitHubUserDO {
    private String name;
    private String email;
    private String blog;
    private String bio;
    private String company;
    private String location;
    @JsonProperty("twitter_username")
    private String twitterUsername;
    private Boolean hireable;
}

File: Gixor-Mobile/app/src/main/java/com/cpy3f2/gixor_mobile/model/entity/GitHubUserDO.kt
package com.cpy3f2.gixor_mobile.model.entity

data class GitHubUserDO(
    val name: String? = null,
    val email: String? = null,
    val blog: String? = null,
    val bio: String? = null,
    val company: String? = null,
    val location: String? = null,

    val twitterUsername: String? = null,
    val hireable: Boolean? = null
)